waku/nwaku.go

```diff
@@ -982,37 +1002,66 @@ func (n *WakuNode) Start() error {
 }
 
 func (n *WakuNode) Stop() error {
-	wg := sync.WaitGroup{}
+	if n == nil {
+		logger := GetLogger()
+		err := errors.New("waku node is nil")
+		logger.Error("Failed to stop", zap.Error(err))
```

Comment:
Related to my previous comments, I think we should be able to do the following:
```suggestion
		err := errors.New("waku node is nil")
		logger.Error("Failed to stop", zap.Error(err))
```

Reply:
I changed the logging to avoid using logger := GetLogger() 
and since logging.go in same package no need for logger. perfix 
also I removed the logger instance of the node config and replaced it with nodename to be added in the logging